src/components/Comments.jsx

```diff
@@ -0,0 +1,27 @@
+import getComments from "../utils/getComments";
+import { useState ,useEffect } from "react";
+import Loading from "./Loading";
+import { useParams } from 'react-router-dom'
+import formatDate from "../utils/FormatDates";
+
+const CommentSection = () => {
+    const [comments,setComments] = useState([]);
+    const  {article_id} = useParams();
+
+    useEffect(() => {
+        getComments(setComments, article_id * 1);
```

Comment:
we should setcomments directly in the useeffect by chaining a .then() to the function and setting the comments in the .then() - therefore we wouldnt need to pass setcomments to the get function